File: ProjetFilRougeFront/src/app/_models/candidate.ts
export class Candidate{
    private _idCandidate: number;
    private _firstname: string;
    private _lastname: string;
    private _email: string;
    private _idLevel: number;
    private _idUser: number;

    constructor(firstname: string, lastname: string, email: string, idLevel: number, idUser: number, id: number = 0){
        this._idCandidate = id;
        this._firstname = firstname;
        this._lastname = lastname;
        this._email = email;
        this._idLevel = idLevel;
        this._idUser = idUser;
    }

    get idCandidate(): number{
        return this._idCandidate;
    }

    get firstname(): string{
        return this._firstname;
    }
    get lastname(): string{
        return this._lastname;
    }
    get email(): string{
        return this._email;
    }
    get idLevel(): number{
        return this._idLevel;
    }
    get idUser(): number{
        return this._idUser;
    }

}

File: ProjetFilRouge/Models/Candidate.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProjetFilRouge.Models
{
    public class Candidate
    {
        public int? idCandidate { get; set; }
        public string firstname { get; set; }
        public string lastname { get; set; }
        public string email { get; set; }
        public int? idUser { get; set; }
        public int? idLevel { get; set; }

        public Candidate(int? idCandidate, string firstname, string lastname, string email, int? idUser, int? idLevel)
        {
            this.idCandidate = idCandidate;
            this.firstname = firstname;
            this.lastname = lastname;
            this.email = email;
            this.idUser = idUser;
            this.idLevel = idLevel;
        }

        public Candidate()
        {
        }
    }
}
